Flow.Launcher.Infrastructure/Hotkey/HotkeyModel.cs

```diff
@@ -86,7 +97,7 @@ private void Parse(string hotkeyString)
                 Ctrl = true;
                 keys.Remove("Ctrl");
             }
-            if (keys.Count > 0)
+            if (keys.Count == 1)
```

Comment:
Will user want to use multiple key as a combo?

Reply:
> Will user want to use multiple key as a combo?

It's not supported. Haven't seen any software supports "ctrl+A+D"

Reply:
Oh we don't support it? I thought this is supported before sorry